Library/Formula/ec2-ami-tools.rb

```diff
@@ -1,7 +1,8 @@
 class Ec2AmiTools < Formula
   homepage "https://aws.amazon.com/developertools/368"
-  url "https://ec2-downloads.s3.amazonaws.com/ec2-ami-tools-1.5.6.zip"
-  sha1 "cb1a0e807e6e4e473a5a462633cb9990ec851093"
+  url "https://ec2-downloads.s3.amazonaws.com/ec2-ami-tools-1.5.7.zip"
+  sha256 "5a45d9f393d2e144124d23d2312b3a8918c5a3f7463b48d55f8db3d56a3fb29f"
+  desc "Command-line utilities to manage Amazon Machine Images (AMI)"
```

Comment:
Please remove the description. This only a temporary situation and only applies on existing formulae. Sorry for the inconvenience :wink: